_posts/2019-07-10-feature-columns/feature_columns.Rmd

```diff
@@ -393,7 +393,7 @@ Let's move on the second application.
 
 To demonstrate the use of `step_crossed_column` to capture interactions, we make use of the `rugged` dataset from Richard McElreath's _rethinking_ package.
 
-We want to predict log GDP based on terrain ruggedness, for a number of countries (170, to be precise). However, the effect of ruggedness is different in Africa as opposed to other continents. Citing from _Statistical Rethinking_
+We want to predict log GDP based on terrain ruggedness, for a number of countries (170, to be precise). However, the effect of ruggedness is different in as opposed to other continents. Citing from _Statistical Rethinking_
```

Comment:
I think here the word "Africa" got lost somehow?

Reply:
Oops, sorry! Made the changes from the GitHUB ui, fixing :P